### PR TITLE
Bunnymark

### PR DESCRIPTION
**Pull request by chippmann**
This adds the bunnymark to the samples with some scores.
I included the `.import` files so one does not have to open the benchmarks in the editor first to be able to run them from commandline.

### CODE REVIEW

File: src/kt_variant.h [hunk 1]
```diff
@@ -236,12 +236,12 @@ namespace ktvariant {
         //TODO: Manage 32 bits systems.
         des->increment_position(encode_uint64(reinterpret_cast<uintptr_t>(ptr), des->get_cursor()));
 
-        String class_name {ptr->get_class()};
+        StringName class_name{ptr->get_class()};
```
Comment (chippmann) [inline src/kt_variant.h:239]: @piiertho Is this still needed?

File: src/kt_variant.h [hunk 2]
```diff
@@ -236,12 +236,12 @@ namespace ktvariant {
...
 
         if (!TypeManager::get_instance().JAVA_ENGINE_TYPES_CONSTRUCTORS.has(class_name)) {
             class_name = ClassDB::get_parent_class(class_name);
 
-            while (class_name.empty()) {
+            while (class_name == String()) {
```
Comment (chippmann) [inline src/kt_variant.h:244]: @piiertho Is this still needed?

File: src/transfer_context.cpp [hunk 3]
```diff
@@ -141,16 +141,11 @@ void TransferContext::icall(JNIEnv* rawEnv, jobject instance, jlong jPtr, jint p
 #endif
 
     write_return_value(buffer, ret_value);
-    jni::JObject local_ref{instance};
-    local_ref.delete_local_ref(env);
```
Comment (chippmann) [inline src/transfer_context.cpp:145]: @piiertho could you also look closely here and the other removals in this file?
Is this change needed or irrelevant after the rebase?

File: src/kt_variant.h (hunk 1 above)
Comment (piiertho) [reply src/kt_variant.h:239]: Yep, it is an optimization. We were creating the `StringName` from the `String` twice.

File: src/kt_variant.h (hunk 2 above)
Comment (piiertho) [reply src/kt_variant.h:244]: As we now directly use a `StringName` yes.

File: src/transfer_context.cpp (hunk 3 above)
Comment (piiertho) [reply src/transfer_context.cpp:145]: All removals are optimization. There is no need to clean local ref as we're in jvm -> cpp case.

**Review by piiertho — APPROVED** [review]

**Review by CedNaru — APPROVED** [review]
